e2e/live/testdata/continue-on-error/Kptfile

```diff
@@ -0,0 +1,11 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: test1
+upstream:
+  type: git
+  git:
+    repo: git@github.com:yuwenma/blueprint-helloworld
```

Comment:
I am curious if there is a helloworld that is not in a particular user repo?  Do we have something we are comfortable with in our own repo?

Reply:
Let's update this in a future PR since it's an irrelevant detail, and I need to get this in to check other fixes.